### DISCUSSION TITLE
Ong Dev Radio

### DISCUSSION THREAD

**Discussion by cutbypham** [OngDev/community#169] [Project Ideas]
Kiểu 1 chỗ để nghe nhạc cho mọi người, xong có vote nhạc đồ luôn
Chạy 24/24
làm trang web rồi syns data trang web đó xuống dưới con bot dưới discord 

**Info**
- domain name: ongdev.cool
- figma design: https://www.figma.com/file/4owsvpC49rt9FW6AhoE2eU/
- figjam whiteboard: https://www.figma.com/file/ulzGLS3ZlUywqLpaayUaFa/

**Final release: https://ongdev.cool/**

**Comment by Shy2003** [comment]
Em thắc mắc làm khó hem ta, em thấy mấy ông nước ngoài ông nào có youtube là mấy ổng để hết, kiểu chỗ để mng ngồi nói chuyện hỏi nhau luôn hay dễ sợ ở VN hem thấy

**Reply by cutbypham** [reply]
cũng okay rứa, chế có thể tạo video request về cái này  :D
